search_username/catalog/models.py

```diff
@@ -0,0 +1,13 @@
+from sqlalchemy import Column, Integer, String
+from sqlalchemy.ext.declarative import declarative_base
+
+Base = declarative_base()
+
+
+class SaveDataBase(Base):
```

Comment:
It's a bad name for the model.
It describes nothing.

Reply:
renamed